core/src/main/java/io/questdb/cairo/DefaultCairoConfiguration.java

```diff
@@ -791,6 +791,11 @@ public int getWalApplyLookAheadTransactionCount() {
         return 20;
     }
 
+    @Override
+    public long getWalApplyMaxTimePerTable() {
```

Comment:
this is "timeQuota" or perhaps "walApplyTableTimeQuotaMs"

Reply:
done